buildenv/jenkins/variables/defaults.yml

```diff
@@ -298,22 +302,17 @@ x86-64_linux:
     11: 'linux-x86_64-normal-server-release'
   node_labels:
     build:
-      all: 'ci.role.build && hw.arch.x86 && sw.os.cent.7'
-      8: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
-      11: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
+      all: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
     docker_image:
       8: 'eclipseopenj9/jenkins-agent-x86-centos6:29'
       11: 'eclipseopenj9/jenkins-agent-x86-centos6:29'
+      all: 'adoptopenjdk/centos7_build_image:latest'
   build_env:
     cmd:
-      all: 'source /home/jenkins/set_gcc_11.2.0_env'
       # use env vars to configure gcc for 8, 11 (v11.2)
-      8: ''
-      11: ''
+      all: ''
     vars:
-      all: 'OPENJ9_JAVA_OPTIONS=-Xdump:system+java:events=systhrow,filter=java/lang/ClassCastException,request=exclusive+prepwalk+preempt'
```

Comment:
@pshipton can we remove these options now?
I see they trace back to #8348 and #8173

Reply:
We should try. Removing the options might cause the failure https://github.com/eclipse-openj9/openj9/issues/8173 to show up again.